cmd/keeper/cmd/keeper.go

```diff
@@ -1588,6 +1589,17 @@ func (p *PostgresKeeper) postgresKeeperSM(pctx context.Context) {
 		if err := pgm.Reload(); err != nil {
 			log.Errorw("failed to reload postgres instance", err)
 		}
+		needsRestart, err := pgm.IsRestartRequired(changedParams)
```

Comment:
Just a not for a future improvement (will prefer doing this in a separate PR): we could report the need restart in the PostgresState reported to the sentinel so it could be showed in the `stolonctl status` output for people that won't enable automatic restart (see below comment).